test/e2e/network/service.go

```diff
@@ -91,7 +91,7 @@ var _ = SIGDescribe("Services", func() {
 		}
 		for _, lb := range serviceLBNames {
 			framework.Logf("cleaning load balancer resource for %s", lb)
-			framework.CleanupServiceResources(cs, lb, framework.TestContext.CloudConfig.Region, framework.TestContext.CloudConfig.Zone)
+			framework.TestContext.CloudConfig.Provider.CleanupServiceResources(cs, lb, framework.TestContext.CloudConfig.Region, framework.TestContext.CloudConfig.Zone)
```

Comment:
I actually changed my mind 😆. I think you were right the first time, let's keep the framework wrappers for these for now and clean it up in a future PR

Reply:
Got it. And I think we can also move this file into `framework/service` and split it in a future PR